crowbar_framework/app/models/node_object.rb

```diff
@@ -1451,6 +1451,31 @@ def disk_release!(device, owner)
     disk_release(device, owner) and save
   end
 
+  def verify_claimed_disks
+    unreferenced_disks = Array.new
+    crowbar_wall[:claimed_disks].each do |disk, _claim|
+      path, device_name = disk.split("/")[-2..-1]
+
+      unless disk =~ /^\/dev\/disk\//
+        unreferenced_disks.push(disk) if node[:block_device][device_name].nil?
+        next
+      end
+
+      if path == "cciss"
```

Comment:
I think this block should move above the ```unless disk =~ /^\/dev\/disk\//``` block. Because that matches ```/dev/cciss/``` disks as well.